### PR TITLE
add ci deployment

### PR DESCRIPTION
**Pull request by cseed**
also remove load balancing from scorecard

### CODE REVIEW

File: ci/Makefile [hunk 1]
```diff
@@ -9,20 +9,15 @@ setup-conda-env:
 update-conda-env:
 	conda env update --name hail-ci -f environment.yml
 
-HAIL_CI_IMAGE_SHORT_NAME = hail-ci
```
Comment (cseed) [inline ci/Makefile]: Not sure if you were using this.  Just seemed unnecessarily noisy and I changed it to match the structure of the other Makefiles.

File: ci/hail-ci-deploy.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -ex
+
+source activate hail-batch
```
Comment (danking) [inline ci/hail-ci-deploy.sh]: hail-ci

File: ci/deployment.yaml.in [hunk 3]
```diff
@@ -55,10 +58,9 @@ metadata:
...
 spec:
   ports:
-    - port: 80
+    - name: http
+      port: 80
       protocol: TCP
       targetPort: 5000
   selector:
     app: hail-ci
-  loadBalancerIP: 104.198.57.66 # hail-ci-0-1
```
Comment (danking) [inline ci/deployment.yaml.in]: I don't follow how DNS is gonna work without static IPs?

Comment (cseed) [reply ci/deployment.yaml.in]: All DNS entries point the nginx reverse proxy now.  This isn't actually being used anymore.

File: ci/hail-ci-deploy.sh (hunk 2 above)
Comment (cseed) [reply ci/hail-ci-deploy.sh]: Ah, good catch, fixed.

**Pull request merged**